components/locid/src/extensions/other/mod.rs

```diff
@@ -19,14 +19,11 @@
 //! let mut loc: Locale = "en-US-a-foo-faa".parse().expect("Parsing failed.");
 //! ```
 
-mod subtag;
-
 use crate::parser::ParserError;
 use crate::parser::SubtagIterator;
 use crate::shortvec::ShortBoxSlice;
+use crate::subtags::Subtag;
```

Comment:
It should still be `pub use` if you want to land this in 1.5, right?

Reply:
I don't want to land it in 1.5